static/templates/settings/notification_settings.hbs

```diff
@@ -69,6 +69,7 @@
                   {{#unless enable_sound_select}}
                   disabled
                   {{/unless}}>
+                    <option value="no sound">no sound</option>
```

Comment:
I'm not sure this is the right label for this; I think `silent` or "Disable sound" might be more natural.  We should also tag the option label for translation.  @alya for thoughts.

Reply:
How about "None"?